core/frontend/src/App.vue

```diff
@@ -50,6 +50,19 @@
             Armed
           </v-card-title>
         </v-card>
+        <v-card
+          v-if="uncalibrated_sensors"
+          v-tooltip="'Some sensors are not calibrated'"
```

Comment:
(optional) perhaps it would be more precise to word it as _"At least one sensor is not calibrated"_ or _"One or more sensors are not calibrated"_.